tasks/image_configuration/downgrade-wpa-hostapd/fetch.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -e
+
+ROOT_DST="${DS_OVERLAY}/tmp/"
+
+if [ "${DS_DISTRO}" == "debian" ] ; then
+	LIBSSL_DEB="libssl1.1_1.1.1w-0+deb11u1_armhf.deb"
+	LIBSSL_SITE="http://ftp.us.debian.org/debian/pool/main/o/openssl/${LIBSSL_DEB}"
+	LIBSSL_SHA256="42130c140f972d938d4b4a5ab9638675e6d1223fcff3042bbcc1829e3646eb00"
+
+	WPA_DEB="wpasupplicant_2.9.0-21_armhf.deb"
+	WPA_SITE="http://ftp.us.debian.org/debian/pool/main/w/wpa/${WPA_DEB}"
+	WPA_SHA256="7bb05cb4f6e3fa3a12d10ea900a4bc7d7d606cacc254115ac28583903b55a2a0"
+
+	HOSTAPD_DEB="hostapd_2.9.0-21_armhf.deb"
+	HOSTAPD_SITE="http://ftp.us.debian.org/debian/pool/main/w/wpa/${HOSTAPD_DEB}"
+	HOSTAPD_SHA256="e9bd2195d497e420dfda9a77726c5bc9f79315faee5633517912e06154fe59c2"
+elif [ "${DS_DISTRO}" == "ubuntu" ] ; then
```

Comment:
Can do.

Question however. Ubuntu is affected in both jammy and lunar. Do you want me to add the DS_RELEASE check for both or just drop that check since we currently only have jammy and lunar?

Reply:
Also, adding this check seem extraneous with the `depends: ...` change to Kconfig. Thoughts?

Reply:
True, but I would still comment what releases the above are for.  Eg, when Debian 13/14 are the norm, its not clear what the above is applicable for without hunting through eventually the kconfigs.